README.txt

```diff
@@ -1,162 +1,89 @@
+INTRODUCTION
+------------
 
-= Vote Up/Down =
+The Vote Up/Down module allows the user to cast votes on some entities in
+a Drupal site with different widgets. This module uses Voting API to implement
+votes and Chaos Tools to provide AJAX support and make custom widgets.
```

Comment:
We may not really need ctools in D8, since a lot of ctools is now part of D8, but that's something we will find out. For now I'm fine with mentioning this.